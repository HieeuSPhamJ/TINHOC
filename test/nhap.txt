With: 4
ls: 
dep: 1
With: 5
ls: 
dep: 1
With: 3
ls: 4 4 
dep: 2
With: 2
ls: 4 
e 4
dep: 2
With: 8
ls: 
dep: 1
With: 10
ls: 
dep: 1
With: 9
ls: 5 
dep: 2
With: 7
ls: 4 5 
dep: 3
With: 12
ls: 
dep: 1
With: 11
ls: 4 
dep: 2
With: 6
ls: 5 4 
dep: 4
With: 1
ls: 3 5 
e 5
dep: 4
0