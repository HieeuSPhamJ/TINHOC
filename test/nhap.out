
                                         .:^^~~!!!!!!!~^:.                                          
                                      .^~~~~~^^^^~~~!!!!77!^.                                       
                                    .^~~^^^^^^^^^^^~~~~~~!77?!.                                     
                                   :~~^^^^^^^^^^^^^^^~~~~~!!7??~                                    
                                  :~~~~^^^^:::::^^^^^^^~~~~!!7?J!                                   
                                 .~~~~~^^^::::::::^^^^^^^~~!!7??J!                                  
                                 ^!~^^^^^::::::::::^~~~~~~~~!77?JY^                                 
                                :~~!~^:^:::::::::::^^^~~~~!!!77??J?.                                
                               ^~::^^::::::^^:^::^^^~!!!!~!!7777??J!                                
                              :~^^~^::^~^^^^^^::^~!7??JJJJJJ777!77??:                               
                              :~!!^::::^~^^^^^^^~!77J5JJY77!!7JJ5PPPJ                               
                              .^^^^~:::^~~^^^^^^~~~~!7??J7^::7PGP5PJ:                               
                               .:::^^::^~~~^^::^^^~~!77!!^^^:~JYYY5:                                
                                .~^^~^:^~~~^^^::::^^~!!!~^^^^^7JJJJ.                                
                                ^~^^~~^~!~^^~^^~^~!!7?7!^^^^:^~?JJ?                                 
                                ^~^^^~~!!~^^^~7~!!~!?77!7777~!7?YY!                                 
                               .~~^^^~!!7!~~~7!~~~?YJ7!77?J55PGP5J.                                 
                               ^!~~^^~777777!7!~~?J??!~^~~?J5PP5Y:                                  
                              :~~~~^^^!7???777!~!!~^~~!!~~!JPPY?.                                   
                             :~~!~~^~~~!7?J?7!!!!!^^~~!7?JYY557                                     
                           .^~~~~~^~~~~!!7?JJJ??7!~^~!!?JY555?                                      
                        .!7??7!!~^^~~~!!77?JJYYYYJ7~!!!7?JJY5^                                      
                      .~?JYY5YYY?7~~~~!!77??YJ??J5P5JJ7JJJJ?7                                       
                   .^!?JJYY5P5Y5P5YJ?7!!!7??JY?7?Y5PP5.....                                         
               .:~7???JJY5YJY5P5PP5YYYYJ?77?JYYJJY5P5.                                              
            .^!?????JJJYYY55YJYPPPPP555555YJJY5YY5PPJ:                                    ..::^:... 
          :~?JJ???JJJYYYYYY555YY5PPPPPPP55PP5555555PBG5J!^.                            ..:::^~^::~!.
        :7JJJJJJJJJJJJJYY55555P555PGPPPPPPP5PPPP555BBBBBBGPJ7!^.         ..           .::^!!^:^~!~~^
      ^7JJJJJJJJJJJJJYYY555555PPPPPGGPPPPPPPPPGGP55B&BBGGGBBBBGP7.      ::^^        .:^^^^~^^~7?~:~:
    ^?JJJ????????JJYYYYYY555PP5PPPPGGGGGPPPPPPPGGPPP##GGGGGGGGGGBP~  ..:::~7.    ..::^~~^:^!!^^~!!: 
  .7JJ???????????JJJYYYYY5555PP5PGPGGGGGGPPPPPPPGGGPG#BGGGGGGGGGGGG7.^^!!~??.  .:::^~^^::~7~^^~??~^:
 .?JJ???????????JJJJJYY5555555PP5PGPGGGGPPPPPPPPPGGGGG#GGGGGGGGGGGGG!^:~J??~ .:^^^~!:::^!!^:~!J?:^^ 
:JJYJ???????????JJJJJYYYY555PPP5PPPPGGGPGPPPPPPPPPPGBG#BGGGGGGGBGGBJ~^^7YJ?::^^^~?~::^~~^::~??~^^:  
JY55????????????JJJJJYYYY5555PPPPGPPGGGGPGPPPPPPPP5PGGBBGGGGGGGGBBP~^^!JY?~^^~^~7~:^!7^::~7J!:^~:   
PYPJ???????????JJJJJJYYY555555PPPGGPPGGGPPGPPPPPPPP5PG#BGGGGGBBGBP!:^~!!^^^^^^^^^^~!!^^~7?7:^~~.    
Y55JJ????????JJJJJJJYYYY555PPPPPPPGG5PGGGPPPPPPPPPPPPPBBGGGGGBBB5!^^^^^^^^^^^^^^^~~^^~?J~^:~7~      
Y55JJJJJ????JJJJJJJYYY55555PPPGGGGGBGPGGGGPPPPPPPPPPPPPGGGBBBBBBY!~~~^^~~^^^^~~^^~~^~??^^~!7^       
JY5YJJJJJJJJJJJJJJYYYYY55PPPPPGGGGGG#PPGGGPPPPPPPPPPPPPGGGBBB##BP7!~~~~^^^^~~~~~!~^~~^:~7?!:        
?YY5YJYJJJJJJJJJJYYYY55555PPGGGGGBBB##PPGGGPPPPGPPPPPPPPGGBB####57~~!~~^^~~~~~~!!^~~^~!??~.         
JYY5YYJJJJJJJJJJYYYYY55555PPPPPPPPGGB&GPGGGPPGGGPPPPGPPPGGBB####5?!!!~~^^~~~~!!~~!7!!7?7^           
Y5555YJJJJJJJJJYYYY555PPPPPPPPP5555PPB#PPGGPPGGGGPPGGPPPPGB###BPJ7!!!~~~~~!!!!!!777777~.            
5P55YJJJJJJJJJJJYYY555555PPPPPPPPPPPPG#BPGGPPGGGGGGGGGGPPGBBG5J7!!!!!!!777!!!!!777??^               
5PPPYJJJJJ??JJYYYYYYYYYYY55YYYYY55PPPPG&BPGPGBGGGGGGGGGGGBGJ?77!!!!!77777!!!7777?JJ?.               
J5PP5JJJ???JJJJJJJJYYYYYYYY55555555PPGG##PPPGBBGGGGGGBBG5J?77?7!!!!!!!7??77?JY5J?JYY:               
?Y5P5JJJJJJJJJJJJJJJJJYYYY55555555PPPGG#BGGBBBBBGGGBBGY?77!7777!!777!7?J?J5PGBPYJJY5!               
^YY55YJJJJJJJJJJJJJJJJJYYY555PPPPPPPGGG#BBBBBBBBBBBGY?77!!!7!!777??77JJPBGGBGG5YJJY5Y.              
 ?YY5YJJJJJJJJJJJJJJJJ?????JJY5PPPPGGGBBBBBBBBBBGPY?77!!!!!!!!7???JJY5G@&GGGGP5YJJY5Y:              